nmtrust

```diff
@@ -9,14 +10,21 @@ usage() {
 Determine if the current NetworkManager connections are trusted.
 
 Options:
-    -f      specify an alternative location for the trusted network file
+    -e      specify an alternative location for the excluded networks file
+    -t      specify an alternative location for the trusted networks file
     -q      be quiet"
 }
 
 file_check() {
+    if [ ! -f "$EXCLUDEFILE" ]; then
```

Comment:
An exclusion file shouldn't be required, so this should only continue if the variable is not empty and the file doesn't exist.

Reply:
On second thought, maybe it is simplest just to default `EXCLUDEFILE` to `/dev/null`. That way nothing needs to change here, and we don't need to take into account the lack of an exclusion file later on when counting the connections or checking for untrusted networks.

Reply:
Would it be ok if the script creates an empty file if the file doesn't exist?

Reply:
Yes, I'm fine with that.

Reply:
Are you ok with using `-e` and `-t` to specify alternative file locations instead of `-f` or do you want to use `-e` and `-f` (or `-e` and `-t` or `-f`) (for backward compatibility).

Reply:
`-e` and `-t` make sense. I'm not worried about backwards compatibility for this.

Reply:
Ok, I changed the file_check